<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="binsy" failures="2" tests="4" name="Test2" time="73.839" errors="0" timestamp="2025-06-18T16:07:44 IST">
  <testcase classname="testScript.LoginTest" name="veriyUserLoginWithValidCredentials" time="0.252">
    <failure type="org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException" message="No valid entries or contents found, this is not a valid OOXML (Office Open XML) file">
      <![CDATA[org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException: No valid entries or contents found, this is not a valid OOXML (Office Open XML) file
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:156)
at org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource.<init>(ZipInputStreamZipEntrySource.java:94)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:132)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:319)
at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:59)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:290)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:286)
at utilities.ExcelUtility.getStringData(ExcelUtility.java:19)
at testScript.LoginTest.veriyUserLoginWithValidCredentials(LoginTest.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
Caused by: java.util.zip.ZipException: Unexpected record signature: 0XA70B0813
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.getNextZipEntry(ZipArchiveInputStream.java:296)
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:152)
... 28 more
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- veriyUserLoginWithValidCredentials -->
  <testcase classname="testScript.LoginTest" name="verifyuserloginwithinvalidusername" time="1.14"/>
  <testcase classname="testScript.LoginTest" name="verifyuserloginwithinvalidpassword" time="0.477"/>
  <testcase classname="testScript.LoginTest" name="verifyuserloginwithinvalidcredential" time="0.0">
    <failure type="org.testng.TestNGException" message="
Method public void testScript.LoginTest.verifyuserloginwithinvalidcredential(java.lang.String,java.lang.String) throws java.io.IOException requires a @DataProvider named :  logindata">
      <![CDATA[org.testng.TestNGException: 
Method public void testScript.LoginTest.verifyuserloginwithinvalidcredential(java.lang.String,java.lang.String) throws java.io.IOException requires a @DataProvider named :  logindata
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- verifyuserloginwithinvalidcredential -->
</testsuite> <!-- Test2 -->
